project/src/molecules/pentane_ua.py

```diff
@@ -0,0 +1,55 @@
+"""Create united-atom representation of pentane."""
+
+import mbuild as mb
+import numpy as np
+
+
+class PentaneUA(mb.Compound):
+    """Create a single particle methane compound."""
+
+    def __init__(self):
+        super(PentaneUA, self).__init__()
+        # Calculate the angle between the two ports
+        angle = np.deg2rad(109.5)
```

Comment:
Should the angle be =CHx-(CH2)-CHy angle i.e. 114 degrees?

Reply:
Sure!